daemon/graphdriver/lcow/remotefs_file.go

```diff
@@ -33,7 +33,7 @@ func (l *lcowfs) OpenFile(path string, flag int, perm os.FileMode) (_ driver.Fil
 	flagStr := strconv.FormatInt(int64(flag), 10)
 	permStr := strconv.FormatUint(uint64(perm), 8)
```

Comment:
Not related to this PR, but wondering why all the conversion is done here, instead of just using the correct verbs in the `fmt.Sprintf()`? (`%d` for int, `%o` for octal (or `%#o` for octal with leading zero); https://play.golang.org/p/9H-rAAA5jaT

Reply:
Defers to @gupta-ak ….

Reply:
Yeah I didn't think about that. All of the `strconv` stuff can be changed to use `%d` or `%o` in the `fmt.Sprintf` statement.